libs/blocks/marketo/marketo.css

```diff
@@ -43,12 +43,20 @@
   display: flex;
   flex-flow: row wrap;
   justify-content: space-between;
+  column-gap: 6%;
```

Comment:
6% might have been too much, 4.6% based on 47.7% width, not sure what that was based on.